api/src/main/java/org/openmrs/module/idgen/IdgenConstants.java

```diff
@@ -37,4 +37,9 @@
 	 * Privilege which grants users permission to upload a batch of identifiers
 	 */
 	public static final String PRIV_UPLOAD_BATCH_OF_IDENTIFIERS = "Upload Batch of Identifiers";
+	
+	/**
+	 * LocationAttribute type for PatientIdentifier Prefixes.
+	 */
+	public static final String PREFIX_LOCATION_ATTRIBUTE_TYPE = "Location Code";
```

Comment:
I think you should move this constant into the LocationBasedPrefixProvider class, but I also think that this should not be hard-coded if this bean is going to exist in this module.  It should be configurable by global property or another mechanism.